xml/en/docs/governance.xml

```diff
@@ -0,0 +1,83 @@
+<!--
+  Copyright (C) F5, Inc.
+  -->
+
+<!DOCTYPE article SYSTEM "../../../dtd/article.dtd">
+
+<article name="NGINX Open Source Governance"
+link="/en/docs/governance.html"
+lang="en"
+rev="1">
+<section>
+
+<para>
+Open source changed the world.
+</para>
+
+<para>
+NGINX is proud to be part of that change, from our very beginning.
+</para>
+
+<para>
+NGINX is committed to continued involvement in open source. To achieve this
+goal, we are making the following commitments that will help the longevity,
+transparency, and impact of our open source projects:
+</para>
+
+</section>
+
+<section name="Our Core Commitments">
+
+<list type="bullet">
+
+<listitem>
+We will be open, consistent, transparent, and fair in our acceptance of
+contributions.
+</listitem>
+
+<listitem>
+We will continue to enhance existing active projects
```

Comment:
either add full stop in the end of the sentence  or remove full stop in similar sentences above/below